Makefile

```diff
@@ -128,10 +128,8 @@ test: codegen fmt vet envtest ## Run unit tests.
 
 test-functional-e2e:
 test-ppnd-e2e:
-test-%: codegen fmt vet envtest ## Run e2e tests. Note we may need to increase the timeout in the future.
-	GOFLAGS="-count=1" go test -timeout 35m -v ./tests/e2e/$* 
-
-
+test-%: envtest ## Run e2e tests. Note we may need to increase the timeout in the future.
+	GOFLAGS="-count=1" ginkgo -v ./tests/e2e/$*
```

Comment:
is there a "timeout" flag for ginkgo call? 

Note that the comment above still refers to a timeout. Also, I should've mentioned it, but in a recent [PR](https://github.com/numaproj/numaplane/pull/570/files), I noticed that we were timing out in 3 different places (here in the Makefile, in the Github action, and in the ginkgo code itself), so I decided to only leave this one and remove the other 2. 

If we can't do it here, then perhaps we should add one of the other ones back.

Reply:
Added timeout here with `ginkgo run`

Reply:
nice